tests/hf/test_integrals.py

```diff
@@ -274,3 +277,142 @@ def test_gradient(symbols, geometry, alpha, coeff):
 
     assert np.allclose(g_alpha, g_ref_alpha)
     assert np.allclose(g_coeff, g_ref_coeff)
+
+
+@pytest.mark.parametrize(
+    ("i", "j", "ri", "rj", "alpha", "beta", "d"),
+    [
+        (
+            0,
+            1,
+            pnp.array([0.0]),
+            pnp.array([20.0]),
+            pnp.array([3.42525091]),
+            pnp.array([3.42525091]),
+            pnp.array([0.0]),
+        ),
+    ],
+)
+def test_diff2(i, j, ri, rj, alpha, beta, d):
+    r"""Test that _diff2 function returns a correct value."""
```

Comment:
My understanding of this test is that it checks if diff_2 is zero for functions with the same center, right? Better to specify this in the description, which is currently too general

Reply:
No, actually it returns zero when the functions are far from each other. Added a description.